wagtail_transfer/views.py

```diff
@@ -97,8 +100,29 @@ def objects_for_export(request):
     }, json_dumps_params={'indent': 2})
 
 
+class UIDField(Field):
```

Comment:
Probably should be a subclass of `ReadOnlyField`

Reply:
Good point - I think several others in the existing api should maybe be using that too then, as I largely copied their approach (children, descendants etc)

Reply:
Fixed now!